castle/cms/browser/controlpanel/announcements.py

```diff
@@ -255,7 +259,7 @@ def handle_export(self, action):
             responsebody = ','.join(fields)
             categories = set()
             if 'form.widgets.export_categories' in self.request.form:
-                if data['export_categories'] != u'':
+                if data['export_categories'] is not None and data['export_categories'] != u'':
```

Comment:
Maybe better to just rewrite to `if data['export_categories']:` or if you really want to be explicit, `if data['export_categories'] not in (None, ''):`

Reply:
Will do